README.md

```diff
@@ -1,31 +1,34 @@
 ---
-description: 'The native NVMe-oF CAS engine for OpenEBS, by OpenEBS'
+description: 'The native NVMe-oF CAS engine for OpenEBS'
```

Comment:
Should it be "of OpenEBS" instead of "for OpenEBS" ?

Reply:
I think you're correct.  Will change.

Reply:
Resolved: 3ca46b8